src/objects/entities/Hedgehog.ts

```diff
@@ -0,0 +1,66 @@
+import * as Phaser from 'phaser';
+import Entity from '@/objects/Entities/Entity';
```

Comment:
change

`import Entity from '@/objects/Entities/Entity';`

to

`import Entity from '@/objects/entities/Entity';`